test/lit-mobx.test.ts

```diff
@@ -24,9 +24,11 @@ import './test-elements';
 
 describe('MobxLitElement', () => {
     it('updates arrays', async () => {
-        const el = await fixture<TestMobxLitElementUpdatesArrays>(html`
-            <test-mobx-lit-element-updates-arrays></test-mobx-lit-element-updates-arrays>
-        `);
+        const el = await fixture<TestMobxLitElementUpdatesArrays>(
```

Comment:
I'm half surprised this work. In _theory_, none of these tests use special bindings meaning if we use the `html` from `@open-wc/testing` in this file we should be able to remove the `as any` casting.

Reply:
Oh I forgot to comment that the tests do not work with this PR. I did not test using `html` from open-wc though. I imagine that open-wc also will make a lit 2 version ”soon enough” so I pushed what I had here as a draft

Reply:
```
> tsc --build tsconfig.json && tsc --build demo/tsconfig.json

node_modules/@lit/reactive-element/reactive-element.d.ts:24:11 - error TS2451: Cannot redeclare block-scoped variable 'JSCompiler_renameProperty'.

24     const JSCompiler_renameProperty: <P extends PropertyKey>(prop: P, _obj: unknown) => P;
             ~~~~~~~~~~~~~~~~~~~~~~~~~

  node_modules/lit-element/lib/updating-element.d.ts:15:9
    15     var JSCompiler_renameProperty: <P extends PropertyKey>(prop: P, _obj: unknown) => P;
               ~~~~~~~~~~~~~~~~~~~~~~~~~
    'JSCompiler_renameProperty' was also declared here.
```

Reply:
Can you try with `html` from there and see what you get? Being we're in NPM with no resolutions, I think it should would, and be resilient to both sides of the `lit@2` release.

Reply:
The `JSCompiler_renameProperty` problem still blocks compilation of the tests

Reply:
I created https://github.com/Polymer/lit-html/issues/1655 for the JSCompiler error